course6/toyapp_ariel/displaylib/CMakeLists.txt

```diff
@@ -4,13 +4,26 @@ add_library(displaylib
 	sdlobject.cc sdlobject.h
 	surface.cc surface.h
         renderingengine.cc renderingengine.h
-        float3.cc float3.h
-        sphere.cc sphere.h
-        ray.cc ray.h
+        float3.h
+        float4.h
+        sphere.h
+        ray.h
         primitive.h
-        intersectioninfo.cc intersectioninfo.h
-        material.cc material.h
+        intersectioninfo.h
+        material.h
+        scene.cc scene.h
+        mat4.cc mat4.h
 )
+
+include(FindOpenMP)
```

Comment:
It's enough to do this kind of stuff in top level CMakeLists file. No need to repeat it in every subproject.